cloud/src/autogluon/cloud/scripts/tabular_serve.py

```diff
@@ -0,0 +1,76 @@
+from autogluon.core.constants import REGRESSION
+from autogluon.core.utils import get_pred_from_proba_df
+from autogluon.tabular import TabularPredictor
+from io import BytesIO, StringIO
+import pandas as pd
+
+
+def model_fn(model_dir):
+    """loads model from previously saved artifact"""
+    model = TabularPredictor.load(model_dir)
+    globals()["column_names"] = model.feature_metadata_in.get_features()
+
+    return model
+
+
+def transform_fn(model, request_body, input_content_type, output_content_type="application/json"):
+
+    if input_content_type == "application/x-parquet":
+        buf = BytesIO(request_body)
+        data = pd.read_parquet(buf)
+
+    elif input_content_type == "text/csv":
+        buf = StringIO(request_body)
+        data = pd.read_csv(buf)
+
+    elif input_content_type == "application/json":
+        buf = StringIO(request_body)
+        data = pd.read_json(buf)
+
+    elif input_content_type == "application/jsonl":
+        buf = StringIO(request_body)
+        data = pd.read_json(buf, orient='records', lines=True)
```

Comment:
Make these subclasses and register in this class as supported extensions similar to converter factory earlier.

Reply:
This is actually a script provided to sagemaker and we need all contents in a single file, so I think the current setup is ok.